### PR TITLE
fix(types) Typehint views.py

### PR DESCRIPTION
**Pull request by fpacifici**

### CODE REVIEW

File: snuba/web/views.py [hunk 1]
```diff
@@ -179,25 +195,25 @@ def handle_invalid_query(exception: InvalidQueryException) -> Response:
...
 @application.route("/snuba/web/static/img/<path:path>")
-def send_img(path):
+def send_img(path: str) -> Any:
     return application.send_static_file(os.path.join("img", path))
 
 
 @application.route("/dashboard")
 @application.route("/dashboard.<fmt>")
-def dashboard(fmt="html"):
+def dashboard(fmt: str = "html") -> Any:
```
Comment (fpacifici) [inline snuba/web/views.py:216]: `application.send_static_file` resolves to Any and not Response.

File: snuba/web/views.py [hunk 2]
```diff
@@ -283,7 +304,7 @@ def _trace_transaction(dataset: Dataset) -> None:
 
 @application.route("/query", methods=["GET", "POST"])
 @util.time_request("query")
-def unqualified_query_view(*, timer: Timer):
+def unqualified_query_view(*, timer: Timer) -> WerkResponse:
```
Comment (fpacifici) [inline snuba/web/views.py:307]: redirect returns a werkzeug response instead of a Flask Response

File: snuba/web/views.py (hunk 1 above)
Comment (lynnagara) [reply snuba/web/views.py:216]: Really? The Mypy revealed type reports this to be Response to me

File: snuba/web/views.py (hunk 2 above)
Comment (lynnagara) [reply snuba/web/views.py:307]: total nitpick, but `WerkzeugResponse` reads better `WerkResponse` to me

Comment (fpacifici) [reply snuba/web/views.py:307]: ok

File: snuba/web/views.py (hunk 1 above)
Comment (fpacifici) [reply snuba/web/views.py]: fixed

File: snuba/web/views.py [hunk 3]
```diff
@@ -262,11 +281,13 @@ def health() -> Response:
...
 
 
-def parse_request_body(http_request):
+def parse_request_body(http_request: Request) -> MutableMapping[str, Any]:
     with sentry_sdk.start_span(description="parse_request_body", op="parse"):
         metrics.timing("http_request_body_length", len(http_request.data))
         try:
-            return json.loads(http_request.data)
+            body = json.loads(http_request.data)
+            assert isinstance(body, dict)
```
Comment (lynnagara) [inline snuba/web/views.py:289]: What's the chances of us hitting this assertion (valid json that isn't a dict)?

Comment (fpacifici) [reply snuba/web/views.py:289]: There is code later that depends on this being a dictionary

**Review by lynnagara — APPROVED** [review]

**Pull request merged**